polaris-for-vscode/.vscode/launch.json

```diff
@@ -0,0 +1,28 @@
+// A launch configuration that compiles the extension and then opens it inside a new window
```

Comment:
I left this in because the dev workflow involves the vscode debugger that gets set up in the `launch.json` and `tasks.json` file. I've been developing locally by having only the `polaris-for-vscode` directory open, otherwise the vscode debugger shortcuts don't work as expected. There's probably a way to set it up so there are tasks in the root `.vscode` file but in my experience the vscode debugger is very fickle and I didn't want to waste time trying to wrangle it together. 

Unless it's a big issue, I'd prefer to leave this as is for now and come back to it if/when it becomes a problem